docs/gateway/b2.md

```diff
@@ -43,7 +43,7 @@ mc ls myb2
 ### Known limitations
 Gateway inherits the following B2 limitations:
 
-- PutObject() does not return an md5sum of the uploaded file as an etag response.  Apps that check vailidity will fail.
+- PutObject S3 API does not return md5sum of the uploaded file as it's etag response. Applications that check validity of the uploaded object by comparing these two values will fail.
```

Comment:
```suggestion
- PutObject S3 API does not return the md5sum of an uploaded file as it's etag response. Applications that check the validity of the uploaded object by comparing these two values will fail.
```

Reply:
We should also mention that this is regardless of `Content-Md5` being set by the client.